plugins/modules/lambda_event.py

```diff
@@ -54,22 +54,28 @@
     type: str
   source_params:
     description:
-      -  Sub-parameters required for event source.
+      - Sub-parameters required for event source.
     suboptions:
       source_arn:
         description:
-          -  The Amazon Resource Name (ARN) of the SQS queue, Kinesis stream or DynamoDB stream that is the event source.
+          - The Amazon Resource Name (ARN) of the SQS queue, Kinesis stream or DynamoDB stream that is the event source.
         type: str
         required: true
       enabled:
         description:
-          -  Indicates whether AWS Lambda should begin polling or readin from the event source.
+          - Indicates whether AWS Lambda should begin polling or readin from the event source.
         default: true
         type: bool
       batch_size:
         description:
-          -  The largest number of records that AWS Lambda will retrieve from your event source at the time of invoking your function.
-        default: 100
+          - The largest number of records that AWS Lambda will retrieve from your event source at the time of invoking your function.
+          - Amazon Kinesis - Default C(100). Max C(10000).
+          - Amazon DynamoDB Streams - Default C(100). Max C(10000).
+          - Amazon Simple Queue Service - Default C(10). For standard queues the max is C(10000). For FIFO queues the max is C(10).
+          - Amazon Managed Streaming for Apache Kafka - Default C(100). Max C(10000).
+          - Self-managed Apache Kafka - Default C(100). Max C(10000).
+          - Amazon MQ (ActiveMQ and RabbitMQ) - Default C(100). Max C(10000).
+          - DocumentDB - Default C(100). Max C(10000).
```

Comment:
```suggestion
          - Amazon Kinesis - Default V(100). Max V(10000).
          - Amazon DynamoDB Streams - Default V(100). Max V(10000).
          - Amazon Simple Queue Service - Default V(10). For standard queues the max is V(10000). For FIFO queues the max is V(10).
          - Amazon Managed Streaming for Apache Kafka - Default V(100). Max V(10000).
          - Self-managed Apache Kafka - Default V(100). Max V(10000).
          - Amazon MQ (ActiveMQ and RabbitMQ) - Default V(100). Max V(10000).
          - DocumentDB - Default V(100). Max V(10000).
```

Reply:
Where's `V()` come from?  last I saw it was supposed to be `C()` for values...